CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+1.14.1 (in progress)
+====================
+
+Bug Fixes
+---------
+
+- Fixed a bug where twilio-video.js was internally using the deprecated
+  RemoteTrack#id property. (JSDK-2173)
```

Comment:
Nitpick: we haven't really been using the #-syntax for referring to methods/properties of classes; instead, we've been saying stuff like

> RemoteTrack's `id` property.